test/CSBondCurve.t.sol

```diff
@@ -66,6 +66,11 @@ contract CSBondCurveTest is Test {
         bondCurve.initialize(_bondCurve);
     }
 
+    function test_getCurveInfo_RevertWhen_InvalidBondCurveId() public {
+        vm.expectRevert(CSBondCurve.InvalidBondCurveId.selector);
+        bondCurve.getCurveInfo(1337);
+    }
+
```

Comment:
Add a positive test also. I mean getting the existing curve. I understand that this is implicitly tested in the other tests, but why not have it here explicitly?